pkg/apb/deprovision.go

```diff
@@ -33,12 +35,17 @@ func Deprovision(instance *ServiceInstance, clusterConfig ClusterConfig, log *lo
 		"deprovision", clusterConfig, instance.Spec,
 		instance.Context, instance.Parameters, log,
 	)
-
 	if err != nil {
 		log.Error("Problem executing apb %s", err)
 		return podName, err
 	}
 
-	log.Info(string(podName))
+	podOutput, err := watchPod(podName, instance.Context.Namespace, log)
+	if err != nil {
+		log.Error("Error returned from watching pod")
+		log.Errorf("error: %s", err.Error())
```

Comment:
Heh, this comes up in a lot of my reviews, it's a preference. I'll update with the first.